src/Admin/AbstractAdmin.php

```diff
@@ -724,6 +724,12 @@ public function getDataSourceIterator()
 
     public function validate(ErrorElement $errorElement, $object)
     {
+        if ('sonata_deprecation_mute' !== (\func_get_args()[0] ?? null)) {
```

Comment:
It's `\func_get_args()[2]`

Reply:
Thanks. Yes, I realized now that index should change with number of arguments.